### DISCUSSION TITLE
I redid my website!

### DISCUSSION THREAD

**Discussion by userexe-me-s-comments** [mrhappyma/site#7] [Blog Comments]
# I redid my website!

And I like actually made it this time not just set up a hugo theme

https://www.userexe.me/posts/new-site

**Comment by mrhappyma** [comment]
Did I fix up comments yet?

**Reply by mrhappyma** [reply]
Yes you did, good work me. How long have those been broken?

**Reply by mrhappyma** [reply]
far too long
